docs/customizing.rst

```diff
@@ -0,0 +1,198 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2020 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Customizing
+===========
+
+
+how to inject/override the Invenio config (url, facets, the DOM id where to render the app): via parametrize or remove the Jinja macro and inject JSON config
+
+Configuring the settings of your Search Application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A helper is provided to generate the configuration needed for the
+React-Searchkit application.
+This configuration contains the following information:
+
+`aggs`: a list of aggregations that will add accordingly facets to your search.
+
+.. code-block:: python
+
+    [
+        {
+            "aggName": "access_right",
+            "field": "access_right",
+            "title": "Access Right"
+        }
+    ]
+
+`appId`: the value here is needed since it will be used internally to
```

Comment:
minor: I wonder how this display. Is it intended to be a list? e.g.:

- aggs
- appId
- ...

Reply:
![Screenshot 2020-12-04 at 17 27 09](https://user-images.githubusercontent.com/16304052/101188407-f8d8a600-3655-11eb-9a61-1a08946437a2.png)